--- po/Makefile.in.in.orig	2005-05-06 12:00:44.000000000 +0200
+++ po/Makefile.in.in	2005-05-06 12:00:44.000000000 +0200
@@ -93,8 +93,8 @@
 
 all: all-@USE_NLS@
 
-all-yes: $(CATALOGS)
-all-no:
+all-yes: $(CATALOGS) Makefile
+all-no: Makefile
 
 $(GETTEXT_PACKAGE).pot: $(POTFILES)
 	$(GENPOT)
@@ -187,6 +187,8 @@
 
 distclean: clean
 	rm -f Makefile Makefile.in POTFILES *.mo *.msg *.cat *.cat.m
+	rm -f po2tbl.sed
+	rm -f .intltool-merge-cache
 
 maintainer-clean: distclean
 	@echo "This command is intended for maintainers to use;"
@@ -202,7 +204,7 @@
 	done
 
 update-po: Makefile
-	$(MAKE) $(GETTEXT_PACKAGE).pot
+	$(GENPOT)
 	tmpdir=`pwd`; \
 	catalogs='$(CATALOGS)'; \
 	for cat in $$catalogs; do \
@@ -245,6 +247,7 @@
 		| sed -e '$$s/\\$$//') > $@-t \
 	    && chmod a-w $@-t \
 	    && mv $@-t $@ )
+	$(MAKE) update-po
 
 Makefile: Makefile.in.in ../config.status POTFILES
 	cd .. \
